### PR TITLE
Closes #108 | Add Shopee Reviews Tagalog dataloader

### LINKED ISSUE #108: Create dataset loader for Shopee Reviews Tagalog
Dataloader name: `shopee_reviews_tagalog/shopee_reviews_tagalog.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?shopee_reviews_tagalog

|     Dataset| shopee_reviews_tagalog |
|-------------|---|
| Description | The Shopee reviews tl 15 dataset is constructed by randomly taking 2100 training samples and 450 samples for testing and validation for each review star from 1 to 5. In total, there are 10500 training samples and 2250 each in validation and testing samples. |
| Subsets | - |
| Languages | tgl, fil |
| Tasks | Text Classification |
| License | Mozilla Public License 2.0 (mpl-2.0) |
| Homepage | https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars |
| HF URL | https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars |
| Paper URL | https://uijrt.com/articles/v4/i8/UIJRTV4I80009.pdf |

### PR DESCRIPTION
**Pull request by ljvmiranda921**
Closes #108

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

**Comment by yongzx** [comment]
Tested successfully via test script and `datasets.load_dataset()`.

File: seacrowd/sea_datasets/shopee_reviews_tagalog/shopee_reviews_tagalog.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+class ShopeeReviewsTagalogDataset(datasets.GeneratorBasedBuilder):
+    """Shopee Reviews Tagalog dataset from https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "text"
+    LABEL_CLASSES = ["0", "1", "2", "3", "4"]
```
Comment (jamesjaya) [inline seacrowd/sea_datasets/shopee_reviews_tagalog/shopee_reviews_tagalog.py:53]: nit I think the labeling may not be very obvious, I would suggest to put an explanation (or maybe just make the label more verbose like "1 star", "2 stars").
```suggestion
    # "N" means N+1 star(s) review, e.g. "2" means 3 stars review
    LABEL_CLASSES = ["0", "1", "2", "3", "4"]
```

**Review by jamesjaya — CHANGES_REQUESTED** [review]
reviewed and tested. I have some nits but apart from that, looks good.

**Comment by ljvmiranda921** [comment]
Updated! Thank you @yongzx and @jamesjaya :)

**Pull request merged**